models/Recording.ts

```diff
@@ -412,7 +412,7 @@ export default function (
           where: {
             type: type,
             processingState: state,
-            processingStartTime: null
+            processing: { [Op.or]: [null, false] }
```

Comment:
Can we make sure that processing is one of the fields returned with a recording by the API?  This will be useful in the UI to show items that are currently processing - at the moment I have an indicator showing that they're queued for processing, but no way of knowing if processing has started yet.

Reply:
yep done